src/api/bank-accounts/bank-account-connection-intents.md

```diff
@@ -0,0 +1,116 @@
+---
+layout:  default
+grand_parent: API Reference
+parent: Bank Accounts
+title: Bank Account Connection Intents
+permalink: /api/bank-account-connection-intents
+---
+
+# Bank Account Connection Intents
+{:.no_toc}
+
+A Bank Account Connection Intent facilitates user authorization of access to Bank Accounts.
+Typically the authorization will follow an OAuth flow where the user is
+redirected to a third-party system to authenticate and select one or more bank
+accounts. After a Bank Account Connection Intent has been authorized, one or more
```

Comment:
This is only true for quartz-accounts, can we make more general or add a caveat

Reply:
Have added that into the section of intent types so it's more specific.